osbs/build/build_request.py

```diff
@@ -649,10 +664,60 @@ def render_import_image(self, use_auth=None):
                 self.dj.dock_json_set_arg('postbuild_plugins', 'import_image',
                                           'insecure_registry', True)
 
+    def render_customizations(self):
+        """
+        Customize prod_inner for site specific customizations
+        """
+
+        disable_plugins = self.customize_conf.get('disable_plugins', [])
+        if len(disable_plugins) == 0:
+            logger.debug("No site-specific plugins to disable")
+        else:
+            for plugin_dict in disable_plugins:
+                try:
+                    self.dj.remove_plugin(
+                        plugin_dict['plugin_type'],
+                        plugin_dict['plugin_name']
+                    )
+                    logger.debug(
+                        "site-specific plugin disabled -> Type:{0} Name:{1}".format(
+                            plugin_dict['plugin_type'],
+                            plugin_dict['plugin_name']
+                        )
+                    )
+                except KeyError:
+                    # Malformed config
+                    logger.debug("Invalid custom configuration found for disable_plugins")
```

Comment:
Does this want to include the `plugin_type` and `plugin_name` it is complaining about?